db/schema.rb

```diff
@@ -301,7 +300,6 @@
               ELSE false
           END AS is_restricted,
       COALESCE(restrictions_cards_points.value, 0) AS eternal_points,
-      COALESCE(restrictions_cards_global_penalty.value, 0) AS global_penalty,
```

Comment:
i'd expect this to be a boolean now instead of a #.  Why remove it altogether from unified_restrictions?